### DISCUSSION TITLE
Is it possible to be a project collaborator?

### DISCUSSION THREAD

**Discussion by stnert** [evilsocket/opensnitch#904] [Q&A]
I would like to help whenever possible with support (issues) here on GitHub. I intend to support the project financially as well.

**Comment by gustavo-iniguez-goya** [comment]
Sure @stnert ! if you think you can help answering questions or debugging issues, please, just comment on the issues or Discussions' questions.

Most of the time the problem is figuring out what the problem is, ask for logs, tests, ... it's not easy for many users to provide all the requested information, so you have to try to reproduce the issue, etc. It's very time consuming.

**Reply by stnert** [reply]
Okay, no problem.

**Comment by stnert** [comment]
Could you please add me as a contributor right away? @gustavo-iniguez-goya

**Reply by gustavo-iniguez-goya** [reply]
nope @stnert , I'm not the owner of the project, I'm a contributor. But you can contribute without that role.
